modules/csg/csg_shape.cpp

```diff
@@ -159,6 +159,74 @@ CSGBrush *CSGShape::_get_brush() {
 	return brush;
 }
 
+int CSGShape::mikktGetNumFaces(const SMikkTSpaceContext *pContext) {
```

Comment:
Reminder that we use `snake_case` for identifiers in Godot's codebase :)
So it should be `mikkt_get_num_faces` and `p_context`.

Reply:
@akien-mga those are the names already used in surface tool but I'm happy to change them

Reply:
I see - it's fine to keep them as is for consistency with `SurfaceTool` then. I guess those were used in ST for easy reference to the thirdparty mikktspace API.